### PR TITLE
Todo list 

### PR DESCRIPTION
**Pull request by massazoum**
Hi team ,
- I used webpack to bundle JavaScript.
-  I use ES6 modules to write modular JavaScript.
- I created an index.html file and write your HTML markup here. Create an empty To Do List placeholder the index.html file must be set as a template using the HTML Webpack Plugin.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+//   // Render each task as a list item
+//   tasks.forEach(task => {
+//     const listItem = document.createElement("li");
+//     listItem.innerHTML =`<div class="divli"><div><input type="checkbox" id="my-checkbox" name="my-checkbox" />
+//     <label for="demoCheckbox"> ${task.description}</div>  <i class="fas fa-ellipsis-v" id='fas'></i>
+//     </div>`  
+//     taskList.appendChild(listItem);
+//   });
+//   container.style.diplay='block';
+// }
```
Comment (leonardodiasb) [inline src/index.js]: - [ ] Kindly remove commented out code. They don't add functionality to your project and can make your files look messy.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  tasks.forEach(task => {
+    const listItem = document.createElement("li");
+    listItem.innerHTML = `<div class="divli"><div><input type="checkbox" id="my-checkbox" name="my-checkbox" />
+    <label for="demoCheckbox"> ${task.description}</div>  <i class="fas fa-ellipsis-v" id='fas'></i>
+    </div>`
+    taskList.appendChild(listItem);
+  });
+
+  container.style.display = 'block';
+};
```
Comment (leonardodiasb) [inline src/index.js]: - [ ] You are rendering the tasks well, but the project required to render in the tasks indexes order. Currently they are being rendered by the order of the position in the array they are stored. Kindly rewrite this function so they can be rendered in the order of the index value.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @massazoum, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional PR description and README file. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @massazoum 👏 👏 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Project set up with Webpack ✔️ 
Has an array of simple to-do tasks ✔️ 
Tasks appear in order of the index values for each task ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**